config/watcher.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: rekor-watcher
+  labels:
+    app: rekor-watcher
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: rekor-watcher
+  template:
+    metadata:
+      labels:
+        app: rekor-watcher
+      annotations:
+        prometheus.io/scrape: "true"
+        prometheus.io/path: /metrics
+        prometheus.io/port: "2112"
+    spec:
+      containers:
+      - name: rekor-watcher
+        image: ko://github.com/sigstore/rekor/cmd/server
+        command: ["/ko-app/server"]
+        ports:
+        - containerPort: 3000
+        - containerPort: 2112 # metrics
+        args: [
+          "watch",
+          "--rekor_server.address=rekor-server",
```

Comment:
Does that not need to be rekor-watcher?

Reply:
Nope! This is how the watcher finds the server. I reused the same flag for consistency, but could also see how that's more confusing...